### PR TITLE
Draft VNC documentation

### PR DESCRIPTION
**Pull request by hdrei**

### CODE REVIEW

**Comment by hdrei** [comment]
Related to issue #111 
Is a new page necessary? Should (some of) the info in this draft-PR be included on the page https://lumi-supercomputer.github.io/LUMI-EasyBuild-docs/l/lumi-vnc/ ?

**Comment by Danzelot** [comment]
I removed the sentence about installing the vnc module as it is preinstalled. I think it is ready to be merged. What do you think, @hdrei ?

File: docs/software/packages/vnc.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+The lumi-VNC modules that are pre-installed on LUMI as EasyBuild recipes provide containerized versions of the [TurboVNC](https://turbovnc.org/About/Introduction) server. These can be used for remote graphics until the OpenOnDemand graphical user interface is available on LUMI.
+
+Connecting with a VNC client such as [TigerVNC](https://tigervnc.org/) or TurboVNC (or via web browser) requires setting up ssh port forwarding from your laptop/desktop to LUMI. Instructions how to do this and which ports to use will be printed when starting the VNC server. 
+
+Don't expect a full-featured desktop environment: The X server runs the fluxbox.org window manager but currently does not support any desktop environment.
+
+
+## Available VNC modules on LUMI
+
+See the versions of VNC available and more detailed information [on the page in the software guides](https://lumi-supercomputer.github.io/LUMI-EasyBuild-docs/l/lumi-vnc/)
```
Comment (hdrei) [inline docs/software/packages/vnc.md]:
```suggestion
See the versions of VNC available and more detailed information [on the page in the software library](https://lumi-supercomputer.github.io/LUMI-EasyBuild-docs/l/lumi-vnc/)
```

**Comment by hdrei** [comment]
Thanks @Danzelot !

We have an issue with the output from `module spider lumi-vnc/20230110`, though, as it is not really up to date:

```
You will need to load all module(s) on any one of the lines below before the "lumi-vnc/20230110" module is available to load.

      LUMI/21.12  partition/C
      LUMI/21.12  partition/G
      LUMI/21.12  partition/L
      init-lumi/0.2
```

**Comment by hdrei** [comment]
Could you @klust also check this (draft-)PR, as you seem to be the author of the lumi-vnc page in software library?

File: docs/software/packages/vnc.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Don't expect a full-featured desktop environment: The X server runs the fluxbox.org window manager but currently does not support any desktop environment.
+
+
+## Available VNC modules on LUMI
+
+See the versions of VNC available and more detailed information [on the page in the software library](https://lumi-supercomputer.github.io/LUMI-EasyBuild-docs/l/lumi-vnc/)
+
+Check which modules are available on the system and get usage information with `module spider lumi-vnc`.
+
```
Comment (hdrei) [inline docs/software/packages/vnc.md:19]:
```suggestion
In a basic case, you do not need to load any modules prior to loading a `lumi-vnc` module. The lumi-vnc modules are pre-installed on the system, and available to load if the `init-lumi/0.2` module is loaded (as it is, by default). 


```

**Comment by hdrei** [comment]
Change in the plans to instead have something like this: https://github.com/Lumi-supercomputer/LUMI-SoftwareStack/pull/93 that just improves the description in Software library. 
Should a VNC page be added in the 'Software guides' list? It would then link directly to the lumi-vnc page in software library

**Comment by olouant** [comment]
What's the status for this PR? Can I close as the documentation should go to the software stack doc?

**Comment by hdrei** [comment]
Yes, sorry, I forgot to close this one. The info that was missing from the software stack doc (or from the `module help lumi-vnc/20230110`) is now added to the software stack doc

**Pull request closed without merging**